src/test/java/racing/domain/CarTest.java

```diff
@@ -1,38 +1,51 @@
 package racing.domain;
 
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class CarTest {
-
-    private Car car;
-
     private static final int DEFAULT_POSITION = 1;
     private static final int DEFAULT_POSITION_FOR_TEST = 3;
-
-    @BeforeEach
-    void setUp() {
-        car = new Car();
-    }
+    private static final String DEFAULT_CAR_NAME = "abc";
 
     @Test
-    @DisplayName("자동차의 초기값이 0으로 설정되는지 테스트")
+    @DisplayName("자동차 위치의 초기값이 1로 설정되는지 테스트")
     void defaultCarPositionTest() {
-        Assertions.assertEquals(DEFAULT_POSITION, car.getPosition());
+        Car car = new Car();
+        assertEquals(DEFAULT_POSITION, car.getPosition());
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"abc", "12345", "a2bc"})
+    @DisplayName("자동차의 이름 세팅 테스트")
+    void initializeCarNameTest(String input) {
+        Car car = new Car(input);
+        assertTrue(car.isSameName(input));
     }
 
     @ParameterizedTest
     @CsvSource(value = {"5:4", "2:3", "4:4"}, delimiter = ':')
     @DisplayName("이동 조건에 부합 하는 경우만 전진하는지 테스트")
-    void moveByCondition(int input, int expected) {
-        car.setPosition(DEFAULT_POSITION_FOR_TEST);
+    void moveByConditionTest(int input, int expected) {
+        Car car = new Car(DEFAULT_CAR_NAME, DEFAULT_POSITION_FOR_TEST);
 
         assertThat(car.moveByCondition(input)).isEqualTo(expected);
     }
-}
+
+    @Test
+    @DisplayName("Car 객체의 copy 여부 확인 테스트")
+    void deepCopyTest() {
+        Car car = new Car("a", 4);
+
+        Car copiedCar = car.copy();
+
+        assertEquals(car, copiedCar);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(copiedCar).isEqualTo(car);
```

Reply:
구글링을 해보니 assertEquals 메소드의 expected, actual의 순서를 헷갈릴 가능성이 있고, 가독성 측면에서도 유용한 assertThat을 더 권장하네요! :0
assertThat으로 피드백 반영해보겠습니다 :)